wasmcloud-test-util/src/lib.rs

```diff
@@ -4,13 +4,65 @@ pub mod provider_test;
 #[cfg(not(target_arch = "wasm32"))]
 pub mod cli;
 
-// re-export testing interface
-pub use wasmcloud_interface_testing as testing;
-
 // re-export regex and nkeys
 pub use nkeys;
 pub use regex;
 
+#[allow(dead_code)]
```

Comment:
Is this necessary? If it is, could you add documentation explaining the reason and maybe a TODO referencing an issue about removing it?

Reply:
It used to be required for some of the automated generation of interfaces but apparently not anymore. :-)  Removed.